detection-rules/attachment_fake_zoom_installer.yml

```diff
@@ -12,37 +12,17 @@ source: |
           or .file_type == "html"
           or .content_type == "text/html"
         )
+        any(ml.logo_detect(file.html_screenshot(.)).brands,
+            .name == "Zoom" and .confidence in ("medium", "high")
+        )
+        and any(ml.nlu_classifier(file.parse_html(.).display_text).entities,
+                .name == "request" and .text =~ "download"
+        )
         and any(file.explode(.),
-                any(ml.logo_detect(file.html_screenshot(..)).brands,
-                    .name == "Zoom" and .confidence in ("medium", "high")
-                )
-                and any(ml.nlu_classifier(file.parse_html(..).display_text).entities,
-                        .name == "request" and .text =~ "download"
-                )
-                and any(.scan.url.urls,
+                any(.scan.url.urls,
                         strings.iends_with(.path, ".exe") and .domain.root_domain not in $org_domains
                 )
         )
-    )
-    or any(attachments,
-           (.file_extension in~ $file_extensions_common_archives)
-           and any(file.explode(.),
-                   (
-                     .file_extension in~ ("html", "htm", "shtml", "dhtml")
-                     or ..file_type == "html"
-                     or ..content_type == "text/html"
-                   )
-                   and any(ml.logo_detect(file.html_screenshot(..)).brands,
```

Comment:
Removed this entirely. We currently can't look at raw files in archives. This one is referencing the archive attachment.